src/demo/demo.jsx

```diff
@@ -143,5 +224,11 @@ render((
 
       <ThumbCarousel contents={carouselData}></ThumbCarousel>
     </div>
+
+    <div>
+      <h3>MoFo Footer. Uses mofo-bootstrap as dependency.</h3>
```

Comment:
I think this was because of the `<select>` at the top needing to be changed. If we're ok getting rid of that dropdown, then getting rid of this text makes sense, but I think that might be a follow up PR?